chipy_org/apps/sponsors/tests.py

```diff
@@ -1,22 +1,23 @@
 # pylint: disable=invalid-name, duplicate-code, redefined-outer-name
 import pytest
 from django.conf import global_settings
-from django.test import Client, override_settings
```

Comment:
Why was 'Client' in this import statement removed as well as in line 18, 19? It seems that the test functions are dependent on the client because they're passed in as a parameter.

Also, I've run the tests and they seem to be passing instead throwing errors so I'm puzzled. Is there something I'm not seeing?